pysilcam/tests/test_process.py

```diff
@@ -19,6 +21,42 @@
 print('MODEL_PATH', MODEL_PATH)
 
 
+@unittest.skipIf((ROOTPATH is None),
+                 "test path not accessible")
+def test_debug_files():
+    '''Testing that the debug images are created'''
+
+    conf_file = os.path.join(ROOTPATH, 'config.ini')
+    conf_file_out = os.path.join(ROOTPATH, 'config_generated.ini')
+    conf = load_config(conf_file)
+
+    data_file = os.path.join(ROOTPATH, 'STN04')
+    conf.set('General', 'loglevel', 'DEBUG')
+    conf.set('General', 'datafile', os.path.join(data_file, 'proc'))
+    conf.set('General', 'logfile', os.path.join(ROOTPATH, 'log.log'))
+    conf.set('ExportParticles', 'outputpath', os.path.join(data_file, 'export'))
+    if MODEL_PATH is not None:
+        conf.set('NNClassify', 'model_path', MODEL_PATH)
+    conf_file_hand = open(conf_file_out, 'w')
+    conf.write(conf_file_hand)
+    conf_file_hand.close()
+
+    # make sure export directory is empty
+    shutil.rmtree(os.path.join(data_file, 'export'), ignore_errors=True)
+
+    # call process function
+    silcam_process(conf_file_out, data_file, multiProcess=True, nbImages=5)
+
+    imc_files = glob.glob(os.path.join(data_file, 'export', '*-IMC*'))
+    assert len(imc_files) == 5, 'unexpected number of IMC files'
+
+    seg_files = glob.glob(os.path.join(data_file, 'export', '*-SEG*'))
```

Comment:
F841: local variable 'seg_files' is assigned to but never used  <!-- Automatically posted by Lintly -->